docs/SUMMARY.md

```diff
@@ -26,7 +26,7 @@
 * [Installing](content/installation/installation.md)
 	* [Compatibility and requirements](content/installation/install_compatibility_requirements.md)
 	* [Installing the operator and driver](content/installation/install_operator_driver.md)
```

Comment:
what about this file?

Reply:
What is the question here? the names here are aligned with the content

Reply:
I'm referring to [this](https://github.com/IBM/ibm-block-csi-driver/blob/v1.8.0/docs/content/installation/install_operator_driver.md) line:
>Install the operator for IBM® block storage CSI driver